src/instructlab/configuration.py

```diff
@@ -173,9 +186,53 @@ def get_default_config():
             taxonomy_base=DEFAULT_TAXONOMY_BASE,
         ),
         serve=_serve(model_path=DEFAULT_MODEL_PATH),
+        train=_train(
+            train_args=TrainingArgs(
+                model_path=DEFAULT_MODEL_REPO,
```

Comment:
We should change this to use [granite-7b-base](https://huggingface.co/ibm-granite/granite-7b-base/tree/main) and `is_padding_free` should be set to true.

Reply:
can't use 7b-base, not public :) switching to 7b-lab

Reply:
Is 7b-lab a base model though? The input model needs to be one that hasn't been trained yet.